doc/sphinx/source/conf.py

```diff
@@ -275,6 +275,7 @@
     # Additional stuff for the LaTeX preamble.
     'preamble':
     r'''
+   \maxdeadcycles=1000
```

Comment:
could you pls add an inline comment as to why and what this does, us in 5 years will be scratching our heads about it 🍺

Reply:
@bouweandela great 😃 But you forgot this one - it's really not obvious what it do 😁